src/components/Select/index.js

```diff
@@ -54,12 +56,34 @@ export default class Select extends PureComponent {
     }
   };
 
+  chipGroupComponent = selected => {
+    const { onSelect, category } = this.props;
+    return (
+      <ChipGroup>
+        {(selected || []).map(currentChip => {
```

Comment:
When `selected` is empty, using .map() will not return `<Chip>` tags. We can omit the conditional for `(selected || [])` and just use `selected.map()`. For further confidence in type verification, we can use PropTypes validation to ensure `selected` is an empty array if a value for it is not passed during component references.

Reply:
@gurbirkalsi I have resolved the same and have added default value of `selected` to `PropTypes`